applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -1025,6 +1046,170 @@ static void send_enqueued_report(struct subscriber *sub)
 	}
 }
 
+static void hogp_out_rep_write_cb(struct bt_hogp *hogp, struct bt_hogp_rep_info *rep, uint8_t err);
+
+static int send_hid_out_report(struct bt_hogp *hogp, const uint8_t *data, size_t size)
+{
+	uint8_t report_id = data[0];
+	struct bt_hogp_rep_info *out_rep = bt_hogp_rep_find(hogp,
+							    BT_HIDS_REPORT_TYPE_OUTPUT,
+							    report_id);
+
+	if (!out_rep) {
+		/* HID output report with given ID is not supported by the peripheral. */
+		return 0;
+	}
+
+	int err = bt_hogp_rep_write_wo_rsp(hogp, out_rep,
```

Comment:
```suggestion
	err = bt_hogp_rep_write_wo_rsp(hogp, out_rep,
	        ....);
	 if (err) {
```

I would prefer to move `int err;` to the beginning of this function scope.

Reply:
The `err` is used only to handle error returned by this single function. I prefer to limit scope for such variable. Still, I may change it if you insist.

Reply:
I won't insist :)

Reply:
I won't insist :)